src/components/general-components/MapCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Card } from 'antd';
+import './general-component-style/map-card.less';
+
+type MapCardProps = {
+  cardHeader: string;
+  cardBody: string;
+};
+
+const MapCard: React.FC<MapCardProps> = (props) => {
+  return (
+    <div>
+      <Card>
+        <p className="card-header">{props.cardHeader}</p>
```

Comment:
Can/should we stick to using the AntD components for text?

Reply:
Yeah this was a temporary thing just to see if the colors looked right and I forgot to change it to Title and Paragraph. Will be working on themes for typography components this week